src/pages/ZNS/ZNS.tsx

```diff
@@ -260,7 +260,7 @@ const ZNS: React.FC<ZNSProps> = () => {
 				/>
 			)}
 			{/* ZNS Content */}
-			{enableBanner && <Raffle />}
+			{enableBanner && <ClaimNFT requireBanner />}
```

Comment:
@domw30 Given time constraints, I think it is better idea to use existing banner flow as claim flow is similar to mint drop flow and just update the click action to launch claim flow.